### PR TITLE
VZ-5722: Define MetricsTrait path to actuator endpoint for Spring variant of Shockshop demo

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

This PR defines MetricsTrait path to actuator endpoint for Spring variant of Shocks-shop demo.

Fixes VZ-5722

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/examples/socks/sock_shop_example_test.go [hunk 1]
```diff
@@ -270,13 +269,48 @@ var _ = t.Describe("Sock Shop test", Label("f:app-lcm.oam",
 							Eventually(appConfigMetricExists, waitTimeout, pollingInterval).Should(BeTrue())
 						},
 					)
+				} else if getVariant() == "spring" {
+					pkg.Concurrently(
+						func() {
+							Eventually(func() bool {
```
Comment (jmaron99) [inline tests/e2e/examples/socks/sock_shop_example_test.go:275]: much like coherenceMetricExists below, perhaps define a method that takes the component name as an arg?

File: tests/e2e/examples/socks/sock_shop_example_test.go [hunk 2]
```diff
@@ -337,24 +371,6 @@ var _ = clusterDump.AfterSuite(func() {
 	}
 })
 
-// isSockShopServiceReady checks if the service is ready
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply tests/e2e/examples/socks/sock_shop_example_test.go:340]: This function is/was not used by the test, so removed that.

**Comment by jmaron99** [comment]
Would just like to see a successful test run (link)

**Review by jmaron99 — APPROVED** [review]
LGTM

**Comment by bhatpmk** [comment]
> Would just like to see a successful test run (link)

Done

**Pull request merged**